ajax/libs/ion-tree-list/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "ion-tree-list",
+  "filename": "ion-tree-list.min.js",
+  "version": "0.0.11",
+  "description": "Ionic directive for displaying nested list items.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fer/ion-tree-list.git"
+  },
+  "keywords": [
+    "ionic",
+    "ion",
+    "tree",
+    "list"
+  ],
+  "author": "fer <fer@ferqwerty.com>",
+  "license": "MIT",
+  "homepage": "https://fer.github.io/ion-tree-list/",
```

Comment:
I think there is a icon which can link to git repo, so I add this as homepage.
@PeterDaveHello Could you help me determine whether this website can be added? Thanks!